blockchain/backup/NFTtheWorld.sol

```diff
@@ -13,46 +15,66 @@ contract NFTtheWorld {
     uint256[] public dropHashes;
 
     struct NFTOwnership {
-        address owner;
-        uint256 nftId;
+        address payable owner;
+        string uri;
         uint256 dropId;
         uint256 dropTime;
         address reservedFor;
+        uint256 weiPrice;
     }
 
+    // For testing reasons this is public
+    // To check if an address is an admin
+    mapping(address => bool) public adminRights;
+    // Dictionary of form <dropHash>: list of NFTOwnerships
     mapping(uint256 => NFTOwnership[]) public nftOwnerships;
-    //Dictionary of form <user address>: <<dropHash>: <number of reserved NFTs>>
+    // Dictionary of form <user address>: <<dropHash>: <number of reserved NFTs>>
     mapping(address => mapping(uint256 => uint256)) public nftReservations;
```

Comment:
I get it, these dropHash mappings try to simulate an "NFTDrop" class via global variables but in a more difficult way. Does Solidity have objects? Are there problems with them?